python/mdvtools/mdvproject.py

```
@@ -1077,6 +1078,166 @@
             print(f"Error in MDVProject.add_datasource : Error adding datasource '{name}': {e}")
             raise  # Re-raise the exception to propagate it to the caller
 
+    def add_datasource_polars(
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)